documentation/smg/14_documentation.md

```diff
@@ -0,0 +1,47 @@
+# Documentation
+
+This document will outline the process surrounding the documentation currently housed in the `horace-docs` repository. It assumes a basic knowledge of reStructured text (reST) format (https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html) especially the subset of the sphinx-doc toolset (https://www.sphinx-doc.org/en/master/)
```

Comment:
Could we split long lines like this onto multiple lines? It makes it easier to comment on/review specific sentences and also means that if we make a small change to part of the paragraph (e.g. fixing a typo) we don't get a diff for the full paragraph.

A good set of rules to follow is "[semantic line breaks](https://sembr.org/)" when it's easy to do so. The gist being: line length limit of 80 characters, always split lines at the end of a sentence and split long sentences on clauses.